src/Service.GraphQLBuilder/Mutations/DeleteMutationBuilder.cs

```diff
@@ -17,13 +18,18 @@ public static class DeleteMutationBuilder
         /// <param name="name">Name of the GraphQL object to generate the delete field for.</param>
         /// <param name="objectTypeDefinitionNode">The GraphQL object type to generate for.</param>
         /// <param name="configEntity">Entity definition</param>
+        /// <param name="dbEntityName">Entity name in runtime config.</param>
+        /// <param name="databaseType">Database type the operation is targetting.</param>
+        /// <param name="returnEntityName">Name of return type entity for mutation.</param>
```

Comment:
I think more context is necessary in the comments to explain why dbEntityName and returnEntityName could be different.

Reply:
Added more context in the explanation.